ultralytics/utils/callbacks/dvc.py

```diff
@@ -24,7 +24,7 @@
 # DVCLive logger instance
 live = None
 _processed_plots = {}
-
+_plots_to_skip = ['confusion_matrix.png', 'confusion_matrix_normalized.png', 'results.png']
```

Comment:
@daavoo I would still keep them tbh. They are a bit different, also user got used to them, etc. Users can decide to switch them back. On the hand we hard code some names and complicate the logger this way.

Reply:
> They are a bit different, also user got used to them, etc. 

Sorry, not sure I follow these points.
Isn't the content the exact same as the difference that we are skipping here a snapshot image ?

> Users can decide to switch them back. 

What do you mean?

Reply:
sorry, as usual my typing-fast-dyslexia, apologies for this to be confusing.

> They are a bit different 

I meant that they are not exactly the same as the ones DVC is producing (e.g. confusion matrix has a bit different view and some people might find it useful).

> Isn't the content the exact same with the difference that we are skipping here snapshot images of something that is logged as an interactive plot ?

almost, but I was not personally comfortable to decide at that point - they are still different a bit AFAIR and I just didn't saw a strong reason to not keep all of the for a while and let users decide on the UX side.

> Users can decide to switch them back.

I meant that they could toggle them on / off on the UX w/o a lot of effort. So, I didn't see value in hard coding very specific optimizations and kept it simple.

Reply:
got it. Dropping for now